### PR TITLE
iframe_fixes

### PR DESCRIPTION
**Pull request by muhammadqamar**

### CODE REVIEW

File: src/containers/Preview/PlaylistPreview/ActivityShared.js [hunk 1]
```diff
@@ -71,16 +71,16 @@ const ActivityShared = (props) => {
...
-      <section className={embed ? 'embed main-page-content preview ' : 'main-page-content preview'}>
+      <section className={embed ? 'embed main-page-content preview iframe-height-resource-shared ' : 'main-page-content preview iframe-height-resource-shared'}>
         {!!embed
           && (
           <Helmet>
             <script src="https://dev.currikistudio.org/api/storage/h5p/h5p-core/js/h5p-resizer.js" charset="UTF-8" />
           </Helmet>
           )}
-        <div className="flex-container previews">
+        <div className="flex-container previews ">
```
Comment (simonebelotti75) [inline src/containers/Preview/PlaylistPreview/ActivityShared.js:81]: Not necessary here

File: src/containers/Preview/PlaylistPreview/LtiPlaylistPreview.js [hunk 2]
```diff
@@ -79,7 +79,7 @@ function LtiPlaylistPreview(props) {
         </div>
       </div>
 
-      <div className="flex-container previews">
+      <div className="flex-container previews ">
```
Comment (simonebelotti75) [inline src/containers/Preview/PlaylistPreview/LtiPlaylistPreview.js:82]: Not necessary here

File: src/containers/Preview/PlaylistPreview/style.scss [hunk 3]
```diff
@@ -669,3 +669,62 @@ img {
...
+        background-color: #fff;
+      }
+    }
+
+    .activity-bg.left-vdo{
+      width:70% !important;
+    }
+
+  }
+}
```
Comment (simonebelotti75) [inline src/containers/Preview/PlaylistPreview/style.scss:730]: Would you please fix all indent and spacing issues here in stylesheet?

**Review by simonebelotti75 — CHANGES_REQUESTED** [review]

File: src/containers/Preview/PlaylistPreview/LtiPlaylistPreview.js (hunk 2 above)
Comment (muhammadqamar) [reply src/containers/Preview/PlaylistPreview/LtiPlaylistPreview.js]: updated

File: src/containers/Preview/PlaylistPreview/style.scss (hunk 3 above)
Comment (muhammadqamar) [reply src/containers/Preview/PlaylistPreview/style.scss]: updated

File: src/containers/Preview/PlaylistPreview/ActivityShared.js (hunk 1 above)
Comment (muhammadqamar) [reply src/containers/Preview/PlaylistPreview/ActivityShared.js]: updated

**Review by simonebelotti75 — APPROVED** [review]

**Pull request merged**